tasks/components/Label.tsx

```diff
@@ -4,13 +4,13 @@ import type { PropsWithChildren } from 'react'
 
 export type LabelProps = PropsWithChildren<{
   name: string,
-  color?: string
+  color?: 'hw-label-1' | 'hw-label-2' | 'hw-label-3' | 'hw-label-blue' | 'hw-label-pink' // extended these colors for more variations
```

Comment:
create a type for all labels and use clear name values with then a mapping in tailwind `tx({'hw-label-pink': color === 'pink'})`

`type ColorTypes = 'label-1' | 'label-2' | 'label-3' | 'blue' | 'pink' // extended these colors for more variations`

```suggestion
  color?: ColorTypes 
```